ui/src/pageLayout/containers/TreeNav.tsx

```diff
@@ -67,132 +74,134 @@ class TreeSidebar extends PureComponent<Props> {
-      >
-        {navItems.map(item => {
-          const linkElement = (className: string): JSX.Element => {
-            if (item.link.type === 'href') {
-              return <a href={item.link.location} className={className} />
+      <OrgSettings>
+        <TreeNav
+          expanded={isExpanded}
+          headerElement={<NavHeader link={orgPrefix} />}
+          userElement={<UserWidget />}
+          onToggleClick={handleToggleNavExpansion}
+          bannerElement={showUpgradeButton ? <CloudUpgradeNavBanner /> : null}
+        >
+          {navItems.map(item => {
+            const linkElement = (className: string): JSX.Element => {
+              if (item.link.type === 'href') {
+                return <a href={item.link.location} className={className} />
+              }
+
+              return <Link to={item.link.location} className={className} />
             }
+            let navItemElement = (
+              <TreeNav.Item
+                key={item.id}
+                id={item.id}
+                testID={item.testID}
+                icon={<Icon glyph={item.icon} />}
+                label={item.label}
+                shortLabel={item.shortLabel}
+                active={getNavItemActivation(
+                  item.activeKeywords,
+                  location.pathname
+                )}
+                linkElement={linkElement}
+              >
+                {Boolean(item.menu) && (
+                  <TreeNav.SubMenu>
+                    {item.menu.map(menuItem => {
+                      const linkElement = (className: string): JSX.Element => {
+                        if (menuItem.link.type === 'href') {
+                          return (
+                            <a
+                              href={menuItem.link.location}
+                              className={className}
+                            />
+                          )
+                        }
 
-            return <Link to={item.link.location} className={className} />
-          }
-          let navItemElement = (
-            <TreeNav.Item
-              key={item.id}
-              id={item.id}
-              testID={item.testID}
-              icon={<Icon glyph={item.icon} />}
-              label={item.label}
-              shortLabel={item.shortLabel}
-              active={getNavItemActivation(
-                item.activeKeywords,
-                location.pathname
-              )}
-              linkElement={linkElement}
-            >
-              {Boolean(item.menu) && (
-                <TreeNav.SubMenu>
-                  {item.menu.map(menuItem => {
-                    const linkElement = (className: string): JSX.Element => {
-                      if (menuItem.link.type === 'href') {
                         return (
-                          <a
-                            href={menuItem.link.location}
+                          <Link
+                            to={menuItem.link.location}
```

Comment:
are any of these links to external resources? we've been having issues with the Link component giving us 404s lately

Reply:
I actually didn't change anything here is except wrap the whole thing inside another React component. The spacing was updated and the diff makes it look like I might have changed an anchor into a Link. But rest assured, there was no change here.

Reply:
oh, this is a weird diff log. DISREGARD